tests/js/spec/views/stream.spec.jsx

```diff
@@ -430,7 +429,7 @@ describe('Stream', function() {
         savedSearchList: [{id: '789', query: 'is:unresolved', name: 'test'}],
       });
 
-      let actual = wrapper.instance().getInitialState();
+      let actual = wrapper.instance().state;
```

Comment:
nit, this could just be `wrapper.state()`

Reply:
👍 will update

Reply:
Hmmm, doesn't work right now, will leave this and try to figure out why later